source/checksums.c

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include <aws/checksums/checksums.h>
+#include <aws/checksums/private/crc_util.h>
+
+void aws_checksums_library_init(struct aws_allocator *allocator) {
+    (void)allocator;
+
+    aws_checksums_crc32_init();
+    aws_checksums_crc64_init();
+}
```

Comment:
Checksum library should call common init as well.

Reply:
Also the cleanup? Plus our standard of static variable to avoid re-initializing it multiple times etc? 🤔

Reply:
added